api/controllers/controller

```diff
@@ -70,4 +70,15 @@ exports.changeVotes = (req, res, next) => {
     }).catch((err) => {
         next(err);
     });
+};
+
+exports.deleteComment = (req, res, next) => {
+    const comment_id = req.params.comment_id;
+    removeComment(comment_id).then((body) => {
+        if(body == comment_id){
+            res.status(204).send();
+        }
+    }).catch((err) => {
+        next(err);
+    });
 };
```

Comment:
Watch out for loose equality here - we should always use strict equality. Maybe use a more descriptive variable name for 'body' as what we're looking at here is the id specifically.